### PR TITLE
Bach: Rename python packages 

### PR DESCRIPTION
**Pull request by jansentom**
Rename package `bach` -> `objectiv-bach`
Rename package `objectiv_bach` -> `objectiv-bach-open-taxonomy`

The module `objectiv_bach` will be renamed to `bach_open_taxonomy` as well.

We use do not use the `objectiv` part of the namespace in the module names. 

Update all build config, usages, docs, README, git worker config etc.
To make the switch:
```
# cd intro root of repo
pip uninstall bach
pip uninstall objectiv-bach
rm bach/*.egg-info -rf
# if pre-merge, does not do any harm after
rm analysis/objectiv_bach/*.egg-info -rf 
# otherwise
rm analysis/bach_open_taxonomy/*.egg-info -rf 
pip install -e bach
pip install -e analysis/bach_open_taxonomy
```

Test, should not give any errors:
```
ipython
>>> import bach_open_taxonomy
```

### CODE REVIEW

File: analysis/bach_open_taxonomy/setup.cfg [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [metadata]
-name = objectiv_bach
-version = attr: objectiv_bach.__version__
-author = Objectiv
-author_email = hendrik@objectiv.io
-description = Objectiv-specific extensions to Bach - Pandas-like DataFrames backed by SQL
+name = objectiv-bach-open-taxonomy
+version = attr: bach_open_taxonomy.__version__
+author = Objectiv B.V.
+author_email = hi@objectiv.io
```
Comment (thijs-obj) [inline analysis/bach_open_taxonomy/setup.cfg:5]: where does this lead?

File: bach/tests/functional/bach/test_data_and_utils.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 from bach import DataFrame, Series
 from bach.types import get_series_type_from_db_dtype
 
-DB_TEST_URL = os.environ.get('OBJ_DB_TEST_URL', 'postgresql://objectiv:@localhost:5432/objectiv')
+DB_TEST_URL = os.environ.get('OBJ_DB_TEST_URL', 'postgresql://objectiv:0bj3ctiv@localhost:5432/objectiv')
```
Comment (thijs-obj) [inline bach/tests/functional/bach/test_data_and_utils.py:19]: hmm :thinking:

File: analysis/bach_open_taxonomy/setup.cfg (hunk 1 above)
Comment (jansentom) [reply analysis/bach_open_taxonomy/setup.cfg:5]: bob & vincent

File: bach/tests/functional/bach/test_data_and_utils.py (hunk 2 above)
Comment (jansentom) [reply bach/tests/functional/bach/test_data_and_utils.py]: not super secret, but unhmm'ed in follow up commit

**Pull request merged**